src/main/resources/application.yaml

```diff
@@ -0,0 +1,42 @@
+server:
+  port: 8081
+spring:
+  application:
+    name: pw-parking-permit
+#----------------------------------------
+# Open API
+#----------------------------------------
+openapi:
+  name: ${spring.application.name}
+  title: ${spring.application.name}
+  version: '@project.version@'
+
+#----------------------------------------
+# Camunda
+#----------------------------------------
+camunda:
+  bpm:
+    client:
+      base-url: http://localhost:8080/engine-rest
+    deployment:
+      processes:
+        - name: Template process
+          tenant: TEMPLATE_NAMESPACE
+          bpmnResourcePattern: classpath*:processmodels/*.bpmn
+          dmnResourcePattern: classpath*:processmodels/*.dmn
+          formResourcePattern: classpath*:processmodels/*.form
+  worker:
+    max:
+      retries: 3
+    retry:
+      timeout: 10
+
+#----------------------------------------
+# Camunda integration settings
+#----------------------------------------
+# Rest client
+integration:
+  camunda:
+    connectTimeout: 5
+    readTimeout: 20
+    url: http://localhost:8080/engine-rest
```

Comment:
Same here

Reply:
From camunda-template. We should change there aswell.